.buildkite/premerge.steps.yaml

```diff
@@ -15,10 +15,11 @@ linux_builders: &linux_builders
     - "environment=production"
     - "permission_set=builder"
     - "platform=linux"
+    - "machine_type=single"
```

Comment:
Be explicit about machine_type.  This should largely hit cached contents since our changes are quite localized.